CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Help and issues
+
+Found a problem or need some help? The following steps may help:
+
+1. Make sure you have gone through the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki). A lot of questions have an answer in there!
+2. Upgrade to the latest version of the plugin and a recent Spigot/Bukkit build for your Minecraft version.
+3. Check your server's logs, there may be warnings/errors/stack straces to help you out with your issue!
+4. Do a quick search through the existing [issues](https://github.com/PyvesB/AdvancedAchievements/issues), you might get lucky!
+5. Simply open a new issue if none of the above help or if you want to suggest a new feature! Help is free, so be polite and friendly! :heart:
+
+## Pull requests
+
+Want to make Advanced Achievements better, faster, stronger? Contributions are more than welcome, open a pull request and share your code! Here are a few pointers to help you out:
+
+1. Feel free to open an [issue](https://github.com/PyvesB/AdvancedAchievements/issues) to get advice or discuss your change beforehand!
+2. Go through the [Developers](https://github.com/PyvesB/AdvancedAchievements/wiki/Developers) guide to get the project up and running and learn how to interact with the plugin.
+3. If you are unsure about how to do something, simply ask by tagging @PyvesB!
+4. Historically the plugin has had poor test coverage, so don't worry too much if you can't add any tests with your pull request. Nevertheless if you want improve maintainability and practice your unit testing skills, please do!
+5. Before submitting your pull request, verify that your code style is somewhat consistent with the rest of the project. You should be good to go! :+1:
```

Comment:
I think it would be quite hard to properly capture a code style in a Wiki article. Either we can purposely be vague or we can ask people to use a defined formatter (for instance [Google's one](https://github.com/google/styleguide), which has formatter settings available for most major IDEs). We could also enforce some rules via a Maven code style plugin.

Reply:
I have added and configured the [formatter-maven-plugin](https://github.com/revelc/formatter-maven-plugin), it seems to work really neatly. You simply need to run `mvn formatter:format` to format your source code with defined settings. 

I could have included it as a goal within Maven build so that it's run automatically before compilation, but I figured out it may annoy some developers to have their code automatically formatted whilst they're still working on the implementation. It would also slightly slow down the build (even though the plugin seems really fast, ~6 seconds on my slow machine, for ~100 Java source files 👍 ).

I will wait until this pull request is merged to actually run the formatter on the current source code, as I would otherwise have to deal with merge conflicts.